.github/workflows/run-shared-tests.yml

```diff
@@ -84,6 +84,20 @@ jobs:
           path: ./funding-service-design-e2e-checks
           token: ${{ steps.generate_token.outputs.token }}
 
+      # Step 2: Fetch tags and find the latest tag
+      - name: Fetch all tags
+        run: |
+          git fetch --tags
+          LATEST_TAG=$(git describe --tags `git rev-list --tags --max-count=1`)
+          echo "Latest tag: $LATEST_TAG"
+
+      # Step 3: Check out the latest tag
+      - name: Checkout latest tag
+        run: |
+          LATEST_TAG=$(git describe --tags `git rev-list --tags --max-count=1`)
+          git checkout $LATEST_TAG
+          echo "Checked out tag: $LATEST_TAG"
+
```

Comment:
Do we want to always take the 'latest' tag, or do we want to agree a tag name that we use for releases we know are good?

Reply:
Totally happy with the concept though, nice job!

Reply:
I added the latest tag because tag creation need to do after series of e2e check most probably after a successful run of the e2e 

also I thought of adding versions from v1.0.0 already created a one

Reply:
https://github.com/communitiesuk/funding-service-design-e2e-checks/tags following are the tags that I have created